ansible/roles/configure/files/helpers/swssconfig_args_update.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
this change is not for the test. It adds permanent configuration for the testbed. To let it survive the reboot I have to modify start.sh